Neos.Neos/Classes/Controller/Service/NodesController.php

```diff
@@ -93,22 +92,9 @@ class NodesController extends ActionController
      */
     public function indexAction($searchTerm = '', array $nodeIdentifiers = [], $workspaceName = 'live', array $dimensions = [], array $nodeTypes = ['Neos.Neos:Document'], NodeInterface $contextNode = null)
     {
-        $searchableNodeTypeNames = [];
-        foreach ($nodeTypes as $nodeTypeName) {
-            if (!$this->nodeTypeManager->hasNodeType($nodeTypeName)) {
-                $this->throwStatus(400, sprintf('Unknown node type "%s"', $nodeTypeName));
```

Comment:
the error code will now change from outside - is that "breaking", as in we catch the 400 in the ui?

I guess now it will be a 500 if invalid nodes are requested.

Reply:
But the previous behaviour was that no results were returned. Is the 500 worse?

Reply:
no i dont think so lets keep it your way ;)